src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -125,9 +117,7 @@ public function panelAction(Request $request, $token)
      */
     public function toolbarAction(Request $request, $token)
     {
-        if (null === $this->profiler) {
-            throw new NotFoundHttpException('The profiler must be enabled.');
-        }
```

Comment:
This one cannot be replaced as it does not disable the profiler after the check!